AdminWebsite/AdminWebsite/Services/UserAccountService.cs

```diff
@@ -129,7 +130,7 @@ private async Task<NewUserResponse> CreateNewUserInAD(ParticipantRequest partici
 
             await _userApiClient.AddUserToGroupAsync(addUserToGroupRequest);
 
-            if (participant.Hearing_role_name == REPRESENTATIVE)
+            if (participant.Hearing_role_name == REPRESENTATIVE || participant.Hearing_role_name == DEFENCE_ADVOCATE)
```

Comment:
@shaed-parkar  could we go by the user role here rather than adding all the hearing roles here. Hearing roles are updated often and we don't really want to change the user account service based on this.

Reply:
we don't know the user role by this point unfortunately. I tried but the user role is determined by the bookings api once created